### ISSUE TITLE
Why does hyper-iqa not seem in the available metric list?

### ISSUE THREAD

**Issue by Imalne** [chaofengc/IQA-PyTorch#66]
It seems that the [DEFAULT_CONFIGS](https://github.com/chaofengc/IQA-PyTorch/blob/08f88508f71eaf02234ebd04aeda091fdc3ab8e7/pyiqa/default_model_configs.py#L8) doesn't include the HYPERIQA. Is this metric incomplete?

**Comment by chaofengc** [comment]
The implementation of the metric follows the original codes and is fully functional. However, as there are currently no pretrained models available, it is not included in the default configs. Unfortunately, due to time constraints, I am unable to address this issue immediately.

If you urgently need to use this metric, you may attempt to modify the codes and train the model on your own. Additionally, we would greatly appreciate it if you could contribute any retrained models to our repository. Thank you for your understanding.

**Comment by chaofengc** [comment]
I found the previously trained HyperIQA model, and has uploaded it. You may now use it same as other metrics:
https://github.com/chaofengc/IQA-PyTorch/blob/a0e2f43dcd375224ac199e2de43661bc15dfe22b/pyiqa/default_model_configs.py#L316-L321

Please note that the model is retrained, and may have a little different performance as the original paper.
